tests/models/test_renderedtifields.py

```diff
@@ -368,30 +369,36 @@ def test_write(self, dag_maker):
         )
 
     @mock.patch.dict(os.environ, {"AIRFLOW_VAR_API_KEY": "secret"})
-    @mock.patch("airflow.utils.log.secrets_masker.redact", autospec=True)
```

Comment:
Why not do it as:

```
    @mock.patch(
        "airflow.sdk.execution_time.secrets_masker.redact"
        if AIRFLOW_V_3_0_PLUS
        else "airflow.utils.log.secrets_masker.mask_secret.redact",
        autospec=True
    )
```

Then you don't have to change the body of the function.

Reply:
That would require me to import `AIRFLOW_V_3_0_PLUS` at the module level which leads to issues like this https://github.com/apache/airflow/actions/runs/13135020938/job/36649053953?pr=46375